### PR TITLE
Save Note Text to database

### PR DESCRIPTION
**Pull request by DongTrieu1998**

### CODE REVIEW

File: fudy/NoteScreen/Component/StickyNote.qml [hunk 1]
```diff
@@ -13,10 +13,12 @@ FramelessWindow {
 	property int index
 	property string noteTitle
 	property bool noteVisible
+	property string noteText
```
Comment (ghost) [inline fudy/NoteScreen/Component/StickyNote.qml]: Please add the default value for all of them.

File: fudy/NoteScreen/Component/TaskItem.qml [hunk 2]
```diff
@@ -18,7 +18,7 @@ Rectangle {
 	signal removeCurrentItem()
 	signal enabledEditted(bool enabled)
 	signal headerEditted(string headerText)
-	signal noteTextEditted()
+	signal noteTextEditted(string noteText)
```
Comment (ghost) [inline fudy/NoteScreen/Component/TaskItem.qml:21]: QString

File: fudy/NoteScreen/Component/StickyNote.qml [hunk 3]
```diff
@@ -13,10 +13,12 @@ FramelessWindow {
...
 	property int xaxis
 	property int yaxis
 
 	signal noteVisibleUpdated(bool visible)
+	signal noteContentUpdated(string noteContent)
```
Comment (ghost) [inline fudy/NoteScreen/Component/StickyNote.qml:21]: QString

File: fudy/NoteScreen/StickNoteModel.cpp [hunk 4]
```diff
@@ -34,6 +33,12 @@ StickNoteModel::StickNoteModel(QObject* parent)
 	}
 }
 
+using FD = FudyProperties;
```
Comment (ghost) [inline fudy/NoteScreen/StickNoteModel.cpp]: I dont think FudyProperties is a long word. FD take me a while to know it is FDProp

File: fudy/NoteScreen/StickNoteModel.cpp [hunk 5]
```diff
@@ -157,69 +157,50 @@ bool StickNoteModel::removeCompletedItems() {
 	qCInfo(F_LOG::sticknoteModel) << "Table" << db_utils::cTableName
 							  << "removed all items successfully";
 
+	queryData();
 	return true;
 }
 
-bool StickNoteModel::updateProperty(QString tableName, QString property, int id, QVariant value) {
+bool StickNoteModel::updateProperty(const QString& tableName, const QString& property, int id, const QVariant& value) {
```
Comment (ghost) [inline fudy/NoteScreen/StickNoteModel.cpp:164]: Please add Todo to refactor them. I think this function is called to much times

**Review by ghost — CHANGES_REQUESTED** [review]

File: fudy/NoteScreen/Component/StickyNote.qml (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/Component/StickyNote.qml]: Default values were added for them.

File: fudy/NoteScreen/Component/TaskItem.qml (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/Component/TaskItem.qml:21]: I think you have a confusion between C++ and QML syntax. Please check again.

File: fudy/NoteScreen/Component/StickyNote.qml (hunk 3 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/Component/StickyNote.qml:21]: I explained above. Please check.

File: fudy/NoteScreen/StickNoteModel.cpp (hunk 4 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/StickNoteModel.cpp]: I updated this code. Thank you.

File: fudy/NoteScreen/StickNoteModel.cpp (hunk 5 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/StickNoteModel.cpp:164]: Of course, thank you.

**Pull request merged**